consensus/vbft/node_utils.go

```diff
@@ -264,6 +279,14 @@ func calcParticipantPeers(cfg *BlockParticipantConfig, chain *vconfig.ChainConfi
 	return peers
 }
 
+func checkCalcEndorserOrCommitter(end int) bool {
```

Comment:
change this function to 'isCalcProposer(end) bool' ?

Reply:
ok